app/src/main/java/com/github/se/orator/MainActivity.kt

```diff
@@ -97,6 +100,14 @@ class MainActivity : ComponentActivity() {
       }
     }
 
+    textToSpeech =
+        TextToSpeech(this) { status ->
+          if (status == TextToSpeech.SUCCESS) {
+            textToSpeech.setSpeechRate(1.7f)
```

Comment:
Maybe we could adjust the speech rate to make it a bit lower. What do you think about this?

Reply:
Thanks for the recommendation Charbel! I have tried different speeds and came to the conclusion that this was the best. It's not so slow that you get bored and yet it's not so fast you can't keep up.
(Feel free to recommend a better value if you find one!!)

Reply:
After considering your perspective, I fully agree with your approach.